x-pack/solutions/security/plugins/security_solution/public/data_view_manager/redux/slices.ts

```diff
@@ -44,6 +45,9 @@ export const sharedDataViewManagerSlice = createSlice({
     setDefaultDataViewId: (state, action: PayloadAction<string>) => {
       state.defaultDataViewId = action.payload;
     },
+    setAlertDataViewId: (state, action: PayloadAction<string>) => {
```

Comment:
See my previous suggestion to update the `setDefaultDataViewId` to `setDataViewIds` (or something like that)